samza-api/src/main/java/org/apache/samza/operators/functions/InitableFunction.java

```diff
@@ -24,16 +24,16 @@
 
 
 /**
- * interface defined to initalize the context of message transformation functions
+ * A function that can be initialized before execution.
  */
 @InterfaceStability.Unstable
 public interface InitableFunction {
 
   /**
-   * Interface method to initialize the context for a specific message transformation function.
+   * Initializes the function before any messages are processed.
    *
```

Comment:
Are there any guarantees we can provide about the order in which init() is invoked if there are multiple InitableFunctions in a StreamApplication?

It'd be nice to inform the users if so.